gbpservice/neutron/services/servicechain/plugins/msc/drivers/simplechain_driver.py

```diff
@@ -55,7 +55,8 @@
 cfg.CONF.register_opts(service_chain_opts, "simplechain")
 
 # Service chain API supported Values
-sc_supported_type = [pconst.LOADBALANCER, pconst.FIREWALL]
+sc_supported_type = [pconst.LOADBALANCER, pconst.LOADBALANCERV2,
+                     pconst.FIREWALL]
```

Comment:
@njagadish I thought you are going to make the decision. So are we going to add the new service_type?